Sources/SwiftPackageListCore/Output Generation/JSON/JSONGenerator.swift

```diff
@@ -15,7 +15,7 @@ struct JSONGenerator: OutputGenerator {
     
     private let jsonEncoder: JSONEncoder = {
         let encoder = JSONEncoder()
-        encoder.outputFormatting = .prettyPrinted
```

Comment:
Please do not get rid of this option. Because this is an `OptionSet` we can specify both options like that: `encoder.outputFormatting = [.prettyPrinted, .sortedKeys]`